### PR TITLE
Reference globalCompositeOperation doc

### PR DESCRIPTION
**Pull request by tdiprima**
@iangilman, does this change work for you?
Fixes openseadragon/openseadragon#1994

### CODE REVIEW

File: src/openseadragon.js [hunk 1]
```diff
@@ -195,8 +195,9 @@
   *
   * @property {String} [compositeOperation=null]
   *     Valid values are 'source-over', 'source-atop', 'source-in', 'source-out',
-  *     'destination-over', 'destination-atop', 'destination-in',
-  *     'destination-out', 'lighter', 'copy' or 'xor'
+  *     'destination-over', 'destination-atop', 'destination-in', 'destination-out',
+  *     'lighter', 'destination', 'copy', 'xor', etc.
```
Comment (iangilman) [inline src/openseadragon.js:199]: You've added `destination` here, but that's not one of the possible modes... perhaps you meant `difference`?

**Review by iangilman — CHANGES_REQUESTED** [review]
Looks great! Just one detail...

File: src/openseadragon.js (hunk 1 above)
Comment (tdiprima) [reply src/openseadragon.js]: Good catch, Ian!  Yes, that's what I meant.

**Review by iangilman — APPROVED** [review]
Awesome, thank you!

**Pull request merged**